frontend/src/components/activity/content/LAThematicArea.vue

```diff
@@ -0,0 +1,64 @@
+<template>
+  <v-list three-line>
+    <v-list-item v-for="option in activityContent.options().items" :key="option.id" class="px-0">
+      <v-list-item-action>
+        <api-checkbox :id="option.id" fieldname="checked" :uri="option._meta.self" />
+      </v-list-item-action>
+      <label class="v-list-item__content align-baseline" :for="option.id">
+        <v-list-item-title>{{ $tc(`activityContent.laThematicArea.entity.option.${option.translateKey}.name`) }}</v-list-item-title>
+        <v-list-item-subtitle>{{ $tc(`activityContent.laThematicArea.entity.option.${option.translateKey}.description`) }}</v-list-item-subtitle>
+      </label>
+    </v-list-item>
+  </v-list>
+</template>
+
+<script>
+
+import ApiCheckbox from '@/components/form/api/ApiCheckbox'
+
+export default {
+  name: 'LAThematicArea',
```

Comment:
To be consistent you should probably call it CA here as well, or call it LA everywhere.

Reply:
I'm not sure I want to call it CA since there is no English translation from J+S. After all, we don't call Jugend+Sport as Youth+Sport (Y+S).